pkg/codeconfig/codeconfig.go

```diff
@@ -110,21 +120,32 @@ func (s *Stack) GetApiSpec(api string) (*openapi3.T, error) {
 	return doc, nil
 }
 
-func (s *Stack) String() string {
-	funcStrings := make([]string, 0)
-
-	for _, f := range s.functions {
-		funcStrings = append(funcStrings, f.String())
+func (c *CodeConfig) ToStack() *stack.Stack {
```

Comment:
@tjholm this is probably wrong, but a start. let me know how I can better do this

Reply:
This is actually pretty close to what I was thinking.

Will add review comments to relevant lines as I go through it.

Reply:
One more thing, it might be good to add an error return on this method as well, to save us refactoring where we're using this method. I know we will eventually return errors in the case where we encounter issues aggregating resources declared by different functions.